src/dbup-sqlserver/SqlTableJournal.cs

```diff
@@ -21,19 +23,29 @@ public class SqlTableJournal : TableJournal
         /// <example>
         /// var journal = new TableJournal("Server=server;Database=database;Trusted_Connection=True", "dbo", "MyVersionTable");
         /// </example>
-        public SqlTableJournal(Func<IConnectionManager> connectionManager, Func<IUpgradeLog> logger, string schema, string table)
-            : base(connectionManager, logger, new SqlServerObjectParser(), schema, table)
+        public SqlTableJournal(Func<IConnectionManager> connectionManager, Func<IUpgradeLog> logger, Func<IHasher> hasher, string schema, string table)
```

Comment:
Hi @ajhall , I noticed the new parameter (hasher) is missing from the summary header. Did you want to update this now or in your next PR?

Reply:
Fixed in my latest revision.